Packages/com.unity.inputsystem/Documentation~/PlayerInput.md

```diff
@@ -1,20 +1,34 @@
-
-The **Player Input** component represents a single player, and that player's associated [Input Actions](Actions.md), whereas the [**Player Input Manager**](PlayerInputManager.md) component handles setups that allow for several concurrent users (for example, player lobbies and split-screen gameplay in a game).
+The Player Input component provides two related but separate features which can be useful in common game scenarios. These are:
 
+- Configuring how [Actions](Actions.md) map to methods or callbacks in the script that controls your player.
+  
+- Handling local multiplayer scenarios such as player lobbies, device filtering, and screen-splitting.
 
 ## The Player Input component
 
-![PlayerInput](Images/PlayerInput.png)
-
+![PlayerInput](Images/PlayerInput.png)<br/>
 *Above, the Player Input component as visible in the inspector.*
 
-Each [`PlayerInput`](../api/UnityEngine.InputSystem.PlayerInput.html) instance represents a separate player or user. You can use multiple `PlayerInput` instances at the same time (although not on the same `GameObject`) to implement local multiplayer setups. The Input System pairs each player to a unique set of Devices that the player uses exclusively, but you can also manually pair Devices in a way that enables two or more players to share a Device (for example, left/right keyboard splits or hot seat use).
+### Connecting actions to methods or callbacks
+
+The **Player Input** component represents a single player, and the connection between that player's associated device, Actions, and methods or callbacks.
+
+You can use a single instance of a Player Input component in a single-player game to set up a mapping between your Input Actions and methods or callbacks in the script that controls your player.
+
+For example, by using the Player Input component, you can set up a mapping between actions such as "Jump" to C# methods in your script such as `public void OnJump()`.
+
+There are a few options for doing exactly how the Player Input component does this, such as using SendMessage, or Unity Events, which is described in more detail below.
+
+### Handling local multiplayer scenarios
+
+You can also use multiple **Player Input** components (each on a separate instance of a prefab) along with the [**Player Input Manager**](PlayerInputManager.md) component to implement local multiplayer features, such as player lobbies, device filtering, and screen-splitting.
+
+In these local multiplayer scenarios, the Player Input component should be on a prefab which the [**Player Input Manager**](PlayerInputManager.md) has a reference to. The **Player Input Manager** then instantiates players as they join the game and pairs each player instance to a unique device that the player uses exclusively (for example, one gamepad for each player). You can also manually pair devices in a way that enables two or more players to share a Device (for example, left/right keyboard splits or hot seat use).
```

Comment:
I think we should say something about how it pairs them.
I think its this (but I'd like Hakan's input as I might not be 100% correct)

The system looks up the 'control scheme' associated with the control that is pressed.
If one is found it makes that the only active control scheme applicable for that player.

It also looks at the specific device that the control was pressed on (e.g. The first gamepad connected) and binds that to be the only device associated with the player.

... then we can explain that the WASD could be in control scheme named 'left keyboard' and cursors in a control scheme named 'right keyboard' and two players would be able to play on that same keyboard device.